Woof/WoofTests/Model/LocalStorage/LocalStorageDeleteTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+final class LocalStorageDeleteTests: XCTestCase {
+    var localStorage: LocalStorage!
+
+    override func setUp() {
+        let storage = LocalStorage()
+        localStorage = storage
+
+        UserDefaults.standard.removePersistentDomain(forName: Bundle.main.bundleIdentifier!)
+    }
+
+    override func tearDown() {
+        localStorage = nil
+        super.tearDown()
+    }
+
+    func testDeleteMethodExistsInAPI() {
+        localStorage.deleteValue(forKey: "key")
+    }
+
+    func testDeleteValueWithExistingKey() {
+        // given
+        let value = "TestValue"
+        let key = "TestKey"
+
+        localStorage.save(value: value, for: key)
+
+        // when
+        localStorage.deleteValue(forKey: key)
+
+        // then
+        let loadedValue = localStorage.loadValue(for: key, as: String?.self)
+
+        XCTAssertNil(loadedValue as Any?)
```

Comment:
Why do you need the cast to `Any?` here?
I thought the storage should give you the casted value?

Reply:
Fixed